src/aere_repl.erl

```diff
@@ -228,7 +255,7 @@ read_file(Filename) ->
         Res -> Res
     end.
 
-read_modules(Modules) ->
+read_modules(Modules, ContractState) ->
```

Comment:
ContractState is unnecessary, as imported modules have no access to the in-repl state anyway

Reply:
Done here 8596c941cc5252319ea6aa3821a3c79037292f2d